src/components/forms/Number.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormControl } from 'patternfly-react';
+import './Dropdown.css';
+import { eventValueHandler } from './utils';
+
+const onKeyDown = e => {
+  if (!((e.keyCode > 95 && e.keyCode < 106) || (e.keyCode > 47 && e.keyCode < 58) || e.keyCode === 8)) {
```

Comment:
please extract keyboard codes to consts, and use meaningful numbers, also I'm pretty sure PF-react has already some of these defined (maybe can import it)